manubot/cite/tests/test_url.py

```diff
@@ -16,8 +16,8 @@ def test_get_url_csl_item_zotero_nyt():
     """
     url = "https://nyti.ms/1NuB0WJ"
     csl_item = get_url_csl_item_zotero(url)
-    assert (
-        csl_item["title"] == "Unraveling the Ties of Altitude, Oxygen and Lung Cancer"
+    assert csl_item["title"].startswith(
+        "Unraveling the Ties of Altitude, Oxygen and Lung Cancer"
```

Comment:
unrelated to this PR, but looks like NYT has changed the HTML metadata at view-source:https://www.nytimes.com/2016/01/26/science/unraveling-the-ties-of-altitude-oxygen-and-lung-cancer.html

```html
<meta data-rh="true" property="og:title" content="Unraveling the Ties of Altitude, Oxygen and Lung Cancer (Published 2016)"/>
```

Possible that Zotero NYT translator will be updated to remove "(Published 2016)" (since it belongs in date not title)